README.md

```diff
@@ -596,9 +595,9 @@ curl -X POST \
   ],
   "options":{
     "layers":[
-      {                                        // <--- THE DESIRED NEW LAYER IS PLACED AT THE TOP OF THE LAYER TREE
-        "operation":{                          //      INDICATING WHERE IN THE DOCUMENT WE WANT THE LAYER PLACED
-          "add":true                           // <--- NEW KEYWORD TO INDICATE AN ADDITION
+      {                                        
+        "add":{                          	    // <--- NEW KEYWORD TO INDICATE AN ADDITION
+          "insertAbove":549	                    // <--- INDICATES THE LAYER SHOULD BE CREATED ABOVE ID 549
```

Comment:
Should this be "insertAbove": {"id": 549}?

Reply:
@alshamma fixed, thanks